### PR TITLE
Feature/#2440 : Created membership card component and Inegrated it to AboutOverviewPageComponent

### PR DESCRIPTION
**Pull request by Basimohd**
#2440 : I've IntegratedCreated membership card component and Inegrated it to AboutOverviewPageComponent. Could you kindly review and confirm if this approach aligns with the desired implementation?

### CODE REVIEW

File: libs/ui/src/lib/membership-card/membership-card.component.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
+:host {
+  display: block;
+
+  .card-item {
+    max-width: 430px;
+    height: 270px;
+    margin-left: auto;
```
Comment (dtslvr) [inline libs/ui/src/lib/membership-card/membership-card.component.scss:7]: I think you can use `mx-auto` in the template

File: libs/ui/src/lib/membership-card/membership-card.component.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    border-radius: 15px;
+  }
+  .card-item-heading {
+    opacity: 0.7;
+    font-size: 13px;
+    margin-bottom: 5px;
+  }
+  .card-item-name {
+    font-size: 18px;
+    line-height: 1;
```
Comment (dtslvr) [inline libs/ui/src/lib/membership-card/membership-card.component.scss:18]: I think you can use `line-height-1` in the template

File: libs/ui/src/lib/membership-card/membership-card.module.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+import { MembershipCardComponent } from './membership-card.component';
+import { GfLogoModule } from '@ghostfolio/ui/logo';
+
+@NgModule({
+  declarations: [MembershipCardComponent],
+  exports: [MembershipCardComponent],
+  imports: [CommonModule, GfLogoModule],
+  schemas: [CUSTOM_ELEMENTS_SCHEMA]
+})
+export class GFMembershipCardModule {}
```
Comment (dtslvr) [inline libs/ui/src/lib/membership-card/membership-card.module.ts:13]: `GfMembershipCardModule`

**Review by dtslvr — CHANGES_REQUESTED** [review]
Nice addition! 👍🏻 I have added some feedback.

**Comment by Basimohd** [comment]
@dtslvr  I have resolved all the concerns outlined in the comments.

**Review by dtslvr — APPROVED** [review]
Great work, @Basimohd! 🙌🏻 

membership

**Comment by Basimohd** [comment]
@dtslvr Thank you!😊 I'm eager to make even more contributions!

**Pull request merged**